daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/ValueEnricher.scala

```diff
@@ -109,7 +109,7 @@ final class ValueEnricher(engine: Engine) {
 
   def enrichNode[Nid](node: GenNode[Nid, ContractId]): Result[GenNode[Nid, ContractId]] =
     node match {
-      case rb @ Node.NodeRollback(_, _) => // reconsider if fields added
+      case rb @ Node.NodeRollback(_) => // reconsider if fields added
```

Comment:
I think we can drop the comment.

Reply:
yup. dropped this _not very helpful_ comment everywhere.